docs/examples.md

```diff
@@ -164,6 +164,39 @@ jobs:
           comment-summary-in-pr: always
 ```
 
+## Getting the results of the action in a later step
+
+Using the `comment-content` output you can get the results of the action in a workflow step.
+
+```yaml
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
This is necessary to get the nice comment output on the PRs.

Reply:
Oh, my goal was to *not* have a comment. Can we instead include the setting to suppress the normal comment logic?

Maybe the workflow author only wants to use a job summary

Reply:
> Can we instead include the setting to suppress the normal comment logic?

Sure, we can remove this permission if that's the case, happy to approve a PR.